File: Software Technologies/Exam Preparation I/Java/src/main/java/projectrider/entity/Project.java
package projectrider.entity;

import org.springframework.stereotype.Controller;

import javax.persistence.*;

@Entity
@Table(name = "projects")
public class Project {
   private Integer id;
   private String title;
   private String description;
   private Integer budget;

   public Project(){
   }


    public Project(String title, String description, Integer budget) {
        this.title = title;
        this.description = description;
        this.budget = budget;
    }

    @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Column(nullable = false)
    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    @Column(nullable = false)
    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Integer getBudget() {
        return budget;
    }

    public void setBudget(Integer budget) {
        this.budget = budget;
    }
}

File: Software Technologies/Exam Preparation I/JavaScript/models/project.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Project = sequelize.define('Project',{
        Title:{
            type: Sequelize.TEXT,
            allowNull: false,
            required: true
        },
        Description:{
            type: Sequelize.TEXT,
            allowNull: false,
            required: true
        },
        Budget:{
            type: Sequelize.INTEGER,
            allowNull: false,
            required: true
        }
    },{
        timestamps:false
    });

    return Project;
};